tailwind.config.js

```diff
@@ -81,8 +81,7 @@ module.exports = {
       screens: {
         "gt-xs": { min: "440px" }, // mobile
         "lt-sm": { max: "767px" }, // less than sm
-        "gt-sm": { min: "767px" },
-        "lt-md": { max: "1023px" }, // less than md
+        "lt-md": { max: "767px" }, // less than md
```

Comment:
This is now same as lt-sm (which although, should've been `max: 640px` in the first place).
Is it better to make custom lt/gt screens or use 2 default breakpoints to set/reset at intervals?

So lets say we want padding at <767px, 
`lt-md: p-4` or `p-4 md:p-0`

Reply:
added changes as discussed over call.